src/test/kotlin/org/rust/ide/inspections/RsCastToBoolInspectionTest.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+import org.rust.lang.utils.RsDiagnostic
+import org.rust.lang.utils.RsDiagnostic.CastAsBoolError.Companion.errorMessage
+
+class RsCastToBoolInspectionTest : RsInspectionsTestBase(RsCastToBoolInspection::class) {
+
+    fun testCastNumToBool() = checkByText("""
```

Comment:
```suggestion
    fun `test cast num to bool`() = checkByText("""
```
We use names with spaces enclosed in backticks for test methods. [See](https://kotlinlang.org/docs/reference/coding-conventions.html#names-for-test-methods)

Reply:
Oops. I used #713 as reference (where this style was not used), as it was listed in the contributing page [here](https://github.com/intellij-rust/intellij-rust/blob/master/CONTRIBUTING.md#contributing). Might want to update those to more recent PRs, to prevent cases like these.